lib/elixir/lib/list.ex

```diff
@@ -1159,6 +1161,25 @@ defmodule List do
     end
   end
 
+  @doc """
+  Returns a new list subtracting `list2` from `list1`.
+  For each element in `list2` the first occurence in `list1` is deleted.
+
+  ## Examples
+
+      iex> List.subtract([1, 2, 3], [1, 2])
+      [3]
+
+      iex> List.subtract([1, 2, 3, 2, 1], [1, 2, 2])
+      [3, 1]
+
+  """
+  @doc since: "1.17.0"
+  @spec subtract(list, list) :: list
+  def subtract(list1, list2) do
+    :lists.subtract(list1, list2)
+  end
```

Comment:
Should this function be inlined by the compiler?

Reply:
I am not sure tbh. afaict the only List functions that are inlined use a counterpart in `:erlang` module and not the `:lists` module.

Reply:
I think we should just use `defdelegate`, which would inline indeed.

Reply:
There are a couple of functions for which we could do that in the List module then, no? For example `flatten/2`